frontend/src/components/Layout.tsx

```diff
@@ -1,24 +1,39 @@
 import { Link, Outlet } from "react-router";
+import {
+  NavigationMenu,
+  NavigationMenuItem,
+  NavigationMenuLink,
+  NavigationMenuList,
+} from "@/components/ui/navigation-menu";
 
 export const Layout = () => {
   return (
     <div>
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/signin">Sign In</Link>
-          </li>
-          <li>
-            <Link to="/signup">Sign Up</Link>
-          </li>
-          <li>
-            <Link to="/recover">Recover Password</Link>
-          </li>
-        </ul>
-      </nav>
+      <NavigationMenu>
```

Comment:
A more scalable approach would be to create an array of `LINKS` and iterate over them to render the navigation. In the event of a new link to be created, the cost to include it is lower. Also it would be prepared to receive the data from an API for example.